lib/Elastica/Response.php

```diff
@@ -78,14 +78,43 @@ public function __construct($responseString, $responseStatus = null)
      */
     public function getError()
     {
-        $message = '';
+        $error = $this->getFullError();
+        if (!$error) {
```

Comment:
In which case will this line match or not match?

Reply:
I copied this verbatim from this commit https://github.com/ruflin/Elastica/commit/99d7cd147c469e20e0c8e550c431712edda4dd19#diff-e1d9f3f0154639bb558fbe6fbe46d323 however it looks like this is to cover the case where the response has no error i.e. https://github.com/ruflin/Elastica/pull/1418/files#diff-b364251d02e70310f3c6e443372397d3R117